src/pages/installing/installing.md

```diff
@@ -230,5 +230,5 @@ cloudctl case launch \
     --inventory clinicalDataAnnotatorOperator \
     --action applyCustomResources \
     --tolerance 1
```

Comment:
I think you need a backslash on line 232.

Reply:
Thanks, added the backslash